### PR TITLE
[Apizr] Docs updated about Apizr integration

### PR DESCRIPTION
**Pull request by JeremyBP**
And finaly, here we are, the docs update final PR :)

Well that one is quite subjective as it depends on what you want to say, where and how.
Also, I'm french, so it may bring some english typos.
Feel free to fix it or ask me to or rephrase everything ^^

Jeremy

### CODE REVIEW

**Comment by JeremyBP** [comment]
Maybe we should say somewhere that Apizr integration works well starting v6.0.0. Just wondering

**Comment by christianhelle** [comment]
@JeremyBP This is a great starting point

> Well that one is quite subjective as it depends on what you want to say, where and how.
> Also, I'm french, so it may bring some english typos.
> Feel free to fix it or ask me to or rephrase everything ^^

My English isn't perfect either, I was born and raised in the Philippines and live in Copenhagen, Denmark. I'm sure the docs are already full of typos and language misuse. At some point, when I can pull myself together, I will check all documentation through some grammar optimization tool like Grammerly

> Maybe we should say somewhere that Apizr integration works well starting v6.0.0. Just wondering

That would make sense. We will have users who learn about Apizr from Refitter and automatically start on the latest version. But it might be different for existing Apizr users, who would learn about Refitter while still on older versions of Apizr.

File: docs/docfx_project/articles/using-the-generated-code.md [hunk 1]
```diff
@@ -234,4 +234,270 @@ public static IServiceCollection ConfigureRefitClients(
 }
 ```
 
-Personally, they I use Refitter is to generate an interface per endpoint, so when generating code for a large and complex API, I might have several interfaces.
\ No newline at end of file
+Personally, they I use Refitter is to generate an interface per endpoint, so when generating code for a large and complex API, I might have several interfaces.
+
+## Apizr
```
Comment (christianhelle) [inline docs/docfx_project/articles/using-the-generated-code.md:239]: Thanks for writing such detailed instructions here ❤️

**Review by christianhelle — APPROVED** [review]
Looks good to me.

I'm fine with merging this as it is, but if you prefer to update the wording so it specifically mentions a minimum Apizr version requirement, then I can wait with merging the pull request.

I can also add the minimum Apizr version requirement texts myself, but that would have to wait until I'm officially back from my holidays 😄

**Comment by christianhelle** [comment]
@all-contributors please add @JeremyBP for docs

**Pull request merged**